examples/portals-2/src/Child.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function Child() {
+  // The click event on this button will bubble up to parent,
+  // because there is no 'onClick' attribute defined
+  return (
+    <div className="modal">
+      <button>Click</button>
+    </div>
+  );
+}
```

Comment:
The portals examples still have a lot of boilerplate. I see that this adds more flexibility for customization, but I suspect it would also end up being super redundant in the case of most of our examples. 🤔

Reply:
You're right, we can move this already to a version that has less boilerplate (same how `state-and-lifecycle-7` works).